phase1-coordinator/src/storage/storage.rs

```diff
@@ -21,6 +21,7 @@ pub enum Locator {
     RoundHeight,
     RoundState(u64),
     RoundFile(u64),
+    /// round_height, chunk_id, contribution_id, some bool - probably "is verified"
```

Comment:
```suggestion
    /// (round_height, chunk_id, contribution_id, is_verified)
```

Reply:
another candidate for named enum fields or a struct I guess